gradle/spec/fixtures/github/contents_java_with_subdir_settings.json

```diff
@@ -0,0 +1,742 @@
+[
+  {
+    "name": "build.gradle",
+    "path": "build.gradle",
+    "sha": "5dbaa74de419bb77b47d054a3374e309aa91eeab",
+    "size": 2354,
+    "url": "https://api.github.com/repos/VoxelGamesLib/dependencies/contents/build.gradle?ref=master",
```

Comment:
Question: where did this fixture come from? Could it be reduced?

It reads like a mix of files from three different repositories:
* https://github.com/VoxelGamesLib/dependencies , a single module maven project
* https://github.com/Samourai-Wallet/samourai-wallet-android , a multimodule groovy gradle project
* https://github.com/togglz/togglz a multimodule maven project

I'd dig if we could reduce the franken-fixture to just the required files. It'd be extra nice if it were from a single functional repository.

Reply:
Yeah great shout, the fixtures where real confusing. Updated to dependabot fixture repos: https://github.com/dependabot-fixtures/